sources/osgDB/DatabasePager.js

```diff
@@ -0,0 +1,367 @@
+/**
+ * @author Jordi Torres
+ */
+
+
+define( [
+    'q',
```

Comment:
use q in lower case because the file is supposed to be like that, but you should use Q in the code